src/qml/Toast.qml

```diff
@@ -127,12 +178,22 @@ Popup {
     if (opacity == 0) {
       toastContent.visible = false;
       toast.close();
+      animationTimer.reset();
+      animationEnabled = false;
     }
   }
 
-  function show(text, type, action_text, action_function) {
+  function show(text, type, action_text, action_function, stop_function, is_animation_enabled) {
```

Comment:
stop_function -> timeout_function

is_animation_enabled -> timeout_feedback

I'd argue here we should have the timeout_feedback *before* the timeout_function.